server/xpub-server/routes.test.js

```diff
@@ -1,9 +1,9 @@
-jest.mock('@pubsweet/db-manager')
+jest.mock('./health')
```

Comment:
If we're mocking `health`, it's worth writing a unit test for it.

Reply:
Good point!